scripts/build.sh

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
 set -e
 
-source ~/.bashrc
```

Comment:
I thought there was a very good reason for this to be here, but I don't remember what it was.